apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -457,33 +466,37 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                 />
             ) }
 
-            <Field.Input
-                ariaLabel="Application access URL"
-                inputType="url"
-                name="accessUrl"
-                label={
-                    t("console:develop.features.applications.forms.generalDetails.fields.accessUrl.label")
-                }
-                required={ isDiscoverable }
-                placeholder={
-                    t("console:develop.features.applications.forms.generalDetails.fields.accessUrl" +
-                        ".placeholder")
-                }
-                value={ accessUrl }
-                readOnly={
-                    !hasRequiredScope || (
-                        readOnly
-                        && applicationConfig.generalSettings.getFieldReadOnlyStatus(
-                            application, "ACCESS_URL"
-                        )
-                    )
-                }
-                maxLength={ 200 }
-                minLength={ 3 }
-                data-testid={ `${ testId }-application-access-url-input` }
-                hint={ t("console:develop.features.applications.forms.generalDetails.fields.accessUrl.hint") }
-                width={ 16 }
-            />
+            { 
+                !isM2MApplication && (
+                    <Field.Input
+                        ariaLabel="Application access URL"
```

Comment:
Shall we use the i18n label value for the ariaLabel too?

Reply:
Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b